### PR TITLE
Document change in behaviour for db.schema.nodeTypeProperties() and db.schema.relTypeProperties()

### PR DESCRIPTION
**Pull request by gem-neo4j**
The column propertyTypes is a LIST<STRING>, currently it outputs runtime types, but in Cypher25 it outputs Cypher Types.

### CODE REVIEW

**Comment by gem-neo4j** [comment]
[update types in db.schema.node/relTypeProperties](https://trello.com/c/apnpyszr/720-update-types-in-dbschemanode-reltypeproperties)

File: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc [hunk 1]
```diff
@@ -16,6 +16,29 @@ New features are added to the language continuously, and occasionally, some feat
...
+label:functionality[]
+label:updated[]
+[source, cypher, role="noheader"]
+----
+CALL db.schema.nodeTypeProperties() YIELD propertyTypes RETURN propertyTypes;
+CALL db.schema.relTypeProperties() YIELD propertyTypes RETURN propertyTypes;
+----
+a|
+The column `propertyTypes` now returns a list of strings representing the possible Cypher Types the given property has.
+For all available Cypher types, see the section on xref::values-and-types/property-structural-constructed.adoc#types-synonyms[types and their synonyms].
```
Comment (JPryce-Aklundh) [inline modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc]:
```suggestion
The column `propertyTypes` returned by the procedures link:{neo4j-docs-base-uri}/operations-manual/{page-version}/procedures/#procedure_db_schema_nodetypeproperties[`db.schema.nodeTypeProperties()`] and link:{neo4j-docs-base-uri}/operations-manual/{page-version}/procedures/#procedure_db_schema_reltypeproperties[`db.schema.relTypeProperties()`] previously returned a list of strings representing the potential Java types for a given property.
It now returns a list of strings representing the possible Cypher Types the given property has.
For all available Cypher types, see the section on xref::values-and-types/property-structural-constructed.adoc#types-synonyms[types and their synonyms].
```

**Review by JPryce-Aklundh — APPROVED** [review]

**Review by loveleif — APPROVED** [review]
Sorry for the delay. Great! Shouldn't the main docs for these procedures also be updated?

**Comment by gem-neo4j** [comment]
@loveleif Procedures for Neo don't really have main docs 😅 Just a very small table: https://neo4j.com/docs/operations-manual/current/reference/procedures/#procedure_db_schema_reltypeproperties doesn't mention what it used to be, so figured no one knows anyway

**Comment by JPryce-Aklundh** [comment]
@gem-neo4j - that will soon change :) (see preview of PR implementing our new tables: https://neo4j-docs-operations-1820.surge.sh/operations-manual/5/procedures/)

**Comment by gem-neo4j** [comment]
@JPryce-Aklundh ah true true, do you think we need to update the description there?

**Comment by JPryce-Aklundh** [comment]
@gem-neo4j no - it doesnt make any reference to java, so I don't see any need to update the actual argument description. I can add an admonition under the table, however (once the main PR is merged) - will card it.

**Pull request merged**